README.md

```diff
@@ -290,6 +290,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Anita de San Miguel](https://github.com/neetnet200)
   - [Anitha subramanian](https://github.com/anithasubramanian19)
   - [Anjali Goswami](https://github.com/anjaligoswami)
+  - [Anjali-gthb](https://github.com/anjali-gthb)
```

Comment:
```suggestion
  - [Anjali](https://github.com/anjali-gthb)
  - [Anjali Goswami](https://github.com/anjaligoswami)
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
@Pradumnasaraf just curious, why does your suggestion edit two line? 🤔

Reply:
@krshkodes When you have 1st name matching in both lines, GitHub does something like that. In the final changes, it will only be one line :)